Dockerfile

```diff
@@ -98,3 +99,23 @@ ENV UBSAN_OPTIONS=print_stacktrace=1
 ENV ASAN_OPTIONS=detect_stack_use_after_return=1:check_initialization_order=1:strict_init_order=1
 RUN ln -s /opt/rocm/llvm/bin/llvm-symbolizer /usr/bin/llvm-symbolizer
 
+#Setup HIPIFY and rocRAND libraries (used for Onnxruntime)
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --allow-unauthenticated \
+    hipfft \
+    rocthrust \
+    rocrand \
+    hipsparse \
+    rccl \
+    rccl-dev \
+    rocm-smi-lib \
+    roctracer-dev \
+    hipcub  \
+    hipblas  \
+    hipify-clang \
+    half && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# add this for roctracer dependancies
+RUN pip3 install CppHeaderParser
+RUN pip3 install packaging==22.0
```

Comment:
Combine this into one line: `pip3 install CppHeaderParser packaging==22.0`.

Reply:
Okay will do

Reply:
Done